src/diffusers/pipelines/marigold/__init__.py

```diff
@@ -0,0 +1,28 @@
+from typing import TYPE_CHECKING
```

Comment:
can we update this file to have the same structure like this https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/pixart_alpha/__init__.py

Reply:
Done